.github/workflows/docker-push-to-registries.yml

```diff
@@ -128,9 +128,16 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: true
           context: ${{ inputs.PATH }}
-          file: ${{ inputs.PATH }}/${{ inputs.DOCKERFILE }}
+          file: ${{ inputs.DOCKERFILE }}
```

Comment:
Spotted elsewhere, it looks for the Dockerfile relative to the `context` so no need to supply the `inputs.PATH` again (and in fact that will actually break builds)